04_numeronym/nicholeL/main_test.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"bytes"
+
+	"github.com/stretchr/testify/assert"
+
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	expected := strconv.Quote("[a11y K8s abc]")
+	actual := strconv.Quote(buf.String())
+	if actual != expected {
```

Comment:
Consider using testify assert (it will shorten this to one line without need for explicit failure message).

Reply:
thanks for your advice